service/src/main/resources/application.yaml

```diff
@@ -96,11 +100,16 @@ springdoc:
     path: /swagger-ui
   writer-with-default-pretty-printer: true
 
+application:
+  kafka:
+    producer:
+      events:
+        topic: ${ENV:folio}.workflow.events
```

Comment:
Again, the environment variable needs to be a more standard environment variable and then more description (such as the example I gave in another PR review `FOLIO_NAMESPACE`).